torchao/_models/llama/generate.py

```diff
@@ -30,7 +31,7 @@ def device_sync(device):
 wd = Path(__file__).parent.parent.resolve()
 sys.path.append(str(wd))
 
-from torchao._models.llama.model import Transformer, prepare_inputs_for_model
+from model import Transformer, prepare_inputs_for_model
```

Comment:
i think we want this to be the same as before, otherwise it can accidentally pull in gpt-fast model and it becomes annoying to test/debug